tests/tests_earth/tests_sensors.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+from astropy import units as u
+from astropy.tests.helper import assert_quantity_allclose
+
+from poliastro.earth.sensors import (
+    max_and_min_ground_range,
+    max_and_min_ground_range_with_specific_azimuth,
+)
+
+
+@pytest.mark.parametrize(
+    "h, n_fov, n_center,expected_Λ_max,expected_Λ_min ",
+    [
+        (
+            800 * u.km,
+            (25 * u.deg).to(u.rad),
+            (40 * u.deg).to(u.rad),
+            0.18736414 * u.rad,
```

Comment:
Do these test cases come from Vallado, or are they hardcoded values?

Reply:
It's the example 11.1 in the Vallado, with all the decimals values.

Reply:
Let's add a comment stating that then!

Reply:
(also for the other cases below)